deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/kubevirt.crd.yaml

```diff
@@ -25,3 +25,7 @@ spec:
     singular: kubevirt
   scope: Namespaced
   version: v1alpha3
+  versions:
```

Comment:
These changes are going to get wiped out when we regenerate.   Components will have to add these